pkg/psmdb/service.go

```diff
@@ -285,6 +288,12 @@ func GetAddr(m *api.PerconaServerMongoDB, pod, replset string) string {
 		":" + strconv.Itoa(int(api.MongodPort(m)))
 }
 
+// GetAddr returns replicaSet pod address in a service mesh
+func GetServiceMeshAddr(m *api.PerconaServerMongoDB, pod, replset string) string {
+	return strings.Join([]string{pod, m.Namespace, m.Spec.ClusterServiceDNSSuffix}, ".") +
+		":" + strconv.Itoa(int(m.Spec.Mongod.Net.Port))
```

Comment:
`Mongod` field is deprecated, please use `api.MongodPort` as in line 288

Reply:
Sorry about that. As you probably can guess this was made using the v1.11.0 branch, and I didn't notice the change in how to fetch the port when merging with main. Fixed now in the latest commit.